cli/cli.rb

```diff
@@ -220,7 +243,9 @@ def start_services
     # this is hokey, but it can take a few seconds for docker daemon to be available
     # TODO: poll in a loop until the docker daemon responds
     sleep 5
-    http_proxy.up(expose_proxy: !!proxy)
+    if proxy
```

Comment:
revert this change -- it's intentional that we still call proxy.up, and just don't bind to the HTTP ports, because the same container also runs the DNS server. the name "proxy" in the code needs to be changed to make that more clear.